src/KafkaFlow.Abstractions/KafkaFlow.Abstractions.csproj

```
@@ -1,15 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
-        <RootNamespace>KafkaFlow</RootNamespace>
-        <PackageId>KafkaFlow.Abstractions</PackageId>
-        <Description>Contains all KafkaFlow extendable interfaces</Description>
-    </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
```

Is there any need to only target net8.0? With this all applications which run on .net46 to .net48 are left behind.

Perhaps best to multitarget.
```suggestion
    <TargetFrameworks>net462;net8.0</TargetFramework>
```

---

Hi @stefannikolei , good question :)

We will stick with netstandard2.0 for now, but please note that ensuring compatibility across all .NET versions is quite challenging... we are doing our best.

https://github.com/Farfetch/kafkaflow/blob/release/4.0/src/KafkaFlow.Abstractions/KafkaFlow.Abstractions.csproj

I will close this MR because we are creating another one: https://github.com/Farfetch/kafkaflow/pull/629